examples/chip-tool/main.cpp

```diff
@@ -275,8 +281,15 @@ void DoEcho(DeviceController::ChipDeviceController * controller, const char * id
             memcpy(buffer->Start(), PAYLOAD, payload_len);
             buffer->SetDataLength(payload_len);
 
-            controller->SendMessage(NULL, buffer);
-            printf("Msg sent to server %s\n", identifier);
+            err = controller->SendMessage(NULL, buffer);
+            if (err)
```

Comment:
My personal preference would be to use a conditional operator here:

    printf("Msg sent to server %s\n", err != CHIP_NO_ERROR ? ErrorStr(err) : identifier)

The above only works because the const message is the same in both cases; your approach is more flexible if the fixed portion of the message changes.

I really like the style of only using booleans as conditionals, as you've done elsewhere in the PR; to that end, even if you choose to stick with the existing approach, I would prefer to see `if (err != CHIP_NO_ERROR)` rather than `if (err)`

Reply:
@vivien-apple ?

Reply:
I took the conditional operator road.